guide/ops/starting-stopping-monitoring.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Starting, Stopping and Monitoring
+layout: website-normal
+---
+
+**NOTE:** This document is for information on starting an Apache Brooklyn
+Server.  For information on using the Brooklyn Client CLI to access an already
+running Brooklyn Server, refer to [Client CLI Reference](cli/index.html).
+
+
+## Starting
+
+To launch Brooklyn, from the directory where Brooklyn is unpacked, run:
+
+{% highlight bash %}
+% bin/brooklyn launch > /dev/null 2>&1 & disown
```

Comment:
It feels like this bit is duplicating http://brooklyn.apache.org/v/latest/ops/server-cli-reference.html

Perhaps we should just reference that instead of having the starting/stopping sub-sections, and ensure that the server-cli-reference is good enough?